packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt

```diff
@@ -2408,15 +2409,13 @@ class HealthPlugin(private var channel: MethodChannel? = null) :
                 binding.addActivityResultListener(this)
                 activity = binding.activity
 
-                if (healthConnectAvailable) {
-                        val requestPermissionActivityContract =
-                                        PermissionController.createRequestPermissionResultContract()
+                val requestPermissionActivityContract =
+                                PermissionController.createRequestPermissionResultContract()
 
-                        healthConnectRequestPermissionsLauncher =
-                                        (activity as ComponentActivity).registerForActivityResult(
-                                                        requestPermissionActivityContract
-                                        ) { granted -> onHealthConnectPermissionCallback(granted) }
-                }
+                healthConnectRequestPermissionsLauncher =
```

Comment:
This needs to always be done now since doing it later (after the user installed Health Connect) is no longer possible.